packages/manager/apps/pci-load-balancer/src/pages/delete/Delete.page.spec.tsx

```diff
@@ -14,6 +16,11 @@ vi.mock('@/api/hook/useLoadBalancer', async () => {
   };
 });
 
+const mockedTrackClick = vi.fn();
+vi.mocked(useOvhTracking).mockReturnValue(({
+  trackClick: mockedTrackClick,
+} as unknown) as ReturnType<typeof useOvhTracking>);
```

Comment:
the cast as unknown shouldn't be required, please fix the typing

Reply:
I want to avoid mocking the all useOvhTracking exports. I'll do something better